apps/beeai-ui/src/modules/runs/components/InputBar.module.scss

```diff
@@ -58,6 +58,12 @@
   column-gap: $spacing-04;
 }
 
+.actionBarLeft {
```

Comment:
Because we are using CSS logical properties in this project, a more proper name would be:
```css
.actionBarStart
```